src/test/scala/com/high-performance-spark-examples/GoldiLocks/EvaluationTests.scala

```diff
@@ -0,0 +1,50 @@
+package com.highperformancespark.examples.goldilocks
+
+import org.apache.spark._
+import org.apache.spark.rdd.RDD
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
+
+
+class EvaluationTests extends FunSuite with BeforeAndAfterAll {
```

Comment:
No I can and should change it.Doesn't need to illustrate anything about the
test so should be the simplest thing possible

On Mon, Jan 25, 2016 at 12:09 AM, Holden Karau <notifications@github.com>
wrote:

> In
> src/test/scala/com/high-performance-spark-examples/GoldiLocks/EvaluationTests.scala
> <https://github.com/high-performance-spark/high-performance-spark-examples/pull/19#discussion_r50663082>
> :
>
> > @@ -0,0 +1,50 @@
> > +package com.highperformancespark.examples.goldilocks
> > +
> > +import org.apache.spark._
> > +import org.apache.spark.rdd.RDD
> > +import org.scalatest.{BeforeAndAfterAll, FunSuite}
> > +
> > +
> > +class EvaluationTests extends FunSuite with BeforeAndAfterAll {
>
> you could also just use SharedSparkContext from
> com.holdenkarau.spark.testing (but if its to illustrate manual testing no
> stress).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/high-performance-spark/high-performance-spark-examples/pull/19/files#r50663082>
> .
>